hw/ip/otbn/dv/otbnsim/sim/edn_client.py

```diff
@@ -62,16 +62,23 @@ def forget(self) -> None:
         self._retry = False
 
     def take_word(self, word: int) -> None:
-        '''Take a 32-bit data word that we've been waiting for'''
-        assert 0 <= word < (1 << 32)
-        assert self._acc is not None
-        assert len(self._acc) < ACC_LEN
-        assert self._cdc_counter is None
+        '''
+        Take a 32-bit data word, which we requested.
 
-        self._acc.append(word)
+        If there is a reset in between an EDN transaction, request flag will
+        drop. In that case, incoming word will be ignored. Otherwise, append
+        it to the internal `_acc` list.
+        '''
+        assert 0 <= word < (1 << 32)
 
-        if len(self._acc) == ACC_LEN:
-            self._cdc_counter = 0
+        if self._acc is None:
+            return
+        else:
+          assert len(self._acc) < ACC_LEN
+          assert self._cdc_counter is None
+          self._acc.append(word)
+          if len(self._acc) == ACC_LEN:
+              self._cdc_counter = 0
```

Comment:
Indentation seems to be wrong here:
```suggestion
            assert len(self._acc) < ACC_LEN
            assert self._cdc_counter is None
            self._acc.append(word)
            if len(self._acc) == ACC_LEN:
                self._cdc_counter = 0
```
Otherwise LGTM!